component/attr_cache/attr_cache_test.go

```diff
@@ -210,8 +207,7 @@ func (suite *attrCacheTestSuite) TestDefault() {
 	defer suite.cleanupTest()
 	suite.assert.Equal(suite.attrCache.Name(), "attr_cache")
 	suite.assert.EqualValues(suite.attrCache.cacheTimeout, 120)
-	suite.assert.Equal(suite.attrCache.cacheOnList, true)
-	suite.assert.Equal(suite.attrCache.noSymlinks, false)
+	// suite.assert.Equal(suite.attrCache.noSymlinks, false)
```

Comment:
Is there a reason we have commented this?

Reply:
after this change this flag is just exported from attr-cache but is not used in attr-cache to make any decision so removed all refrences of it. Kept the flag as is because azstorage component refers this as well.